gobblin-compaction/src/main/java/gobblin/compaction/hivebasedconstructs/MRCompactionEntity.java

```diff
@@ -0,0 +1,23 @@
+package gobblin.compaction.hivebasedconstructs;
+
+import java.util.Properties;
+import lombok.Getter;
+
+
+/**
+ * Entity that stores information required for launching an {@link gobblin.compaction.mapreduce.MRCompactor} job
+ */
+@Getter
+public class MRCompactionEntity {
+  private final String primaryKey;
```

Comment:
Can you be more descriptive what these strings are? Are these column names? If yes, isn't it possible to have more than one primary key and/or delta column?